<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MP3Organizer</class>
 <widget class="QWidget" name="MP3Organizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generador Listas M3U</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Direcci√≥n carpeta:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtPath">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browseButton">
       <property name="text">
        <string>Explorar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Archivos MP3 encontrados:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listMP3"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Archivos M3U encontrados / generados:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listM3U"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblResult">
     <property name="text">
      <string>[+]Estado:...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="continueButton">
       <property name="text">
        <string>Generar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
